### DISCUSSION TITLE
Weird production build behavior when bundling with Vite

### DISCUSSION THREAD

**Discussion by jonathanpmartins** [quasarframework/quasar#12778] [General - Components / Directives / etc]
After the upgrade to 2.6.0 vite generated lots of files when bundling things to production. My first load got from 21 requests to 257 requests. My guess is that it is eager loading everything. All quasar components are loaded in separated files. Even moment-js was loaded in a different connection.

My build target:
```
target: {
  browser: ['es2020', 'edge88', 'firefox78', 'chrome87', 'safari14'],
  node: 'node16',
},
```

I need es2020 because I use BigInt.

With 2.5.5 (webpack):
![Screenshot from 2022-03-14 03-10-49](https://user-images.githubusercontent.com/6137992/158115010-44527ddf-385f-47f4-9844-02b3f1338479.png)

With 2.6.0 (vite):
![Screenshot from 2022-03-14 03-10-21](https://user-images.githubusercontent.com/6137992/158114997-7f13fc18-a5c9-41f1-a65b-e2317aaf26c5.png)

Can I mess with some configuration to reduce the number of files generates in the production build?
Is this the default behavior?

**Comment by suleimanas** [comment]
maybe this can help: https://rollupjs.org/guide/en/#outputmanualchunks

**Comment by tetra12** [comment]
Same here for Quasar 2.7.5 and Vite 2.9.13. 
Using `BigInt` and `es2020` configs yields in build error
